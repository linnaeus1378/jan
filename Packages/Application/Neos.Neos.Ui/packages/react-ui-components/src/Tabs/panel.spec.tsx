import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Panel, {PanelProps} from './panel';

describe('<Panel/>', () => {
    const props: PanelProps = {
        title: 'TitleString',
        theme: {
            panel: 'panelBaseClassName',
        },
        children: 'Foo children',
    };

    it('should render correctly.', () => {
        const wrapper = shallow(<Panel {...props}/>);

        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should have a "displayName" of "Panel".', () => {
        expect(Panel.displayName).toBe('Panel');
    });
});
