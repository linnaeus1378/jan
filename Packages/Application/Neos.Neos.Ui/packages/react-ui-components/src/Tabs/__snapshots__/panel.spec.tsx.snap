// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Panel/> should render correctly. 1`] = `
<div
  className="panelBaseClassName"
>
  Foo children
</div>
`;
